### PR TITLE
R datasets in observations

### PR DESCRIPTION
**Pull request by Arvinds-ds**
Have created `observations.r` and `tests` in this pull request. Kindly review this. I am not checking in src folder for now as the generation code is python 3 only and is kind of sphagetti code.

### CODE REVIEW

File: observations/r/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2308 @@
...
+from observations.r.uk_driver_deaths import uk_driver_deaths
+from observations.r.uk_gas import uk_gas
+from observations.r.us_acc_deaths import us_acc_deaths
+from observations.r.us_arrests import us_arrests
+from observations.r.us_judge_ratings import us_judge_ratings
+from observations.r.us_personal_expend import us_personal_expend
+from observations.r.va_deaths import va_deaths
+from observations.r.www_usage import www_usage
+from observations.r.world_phones import world_phones
+from observations.r.airmiles import airmiles
```
Comment (dustinvtran) [inline observations/r/__init__.py]: Is there a way to sort these alphabetically in your generation script? Same goes for writing to the `allowed_symbols` list.

**Review by dustinvtran — COMMENTED** [review]
Great work! Happy to merge this. Only one request (see comment below).

Related to #19.

File: observations/r/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2308 @@
...
+    'saxony',
+    'sexual_fun',
+    'space_shuttle',
+    'suicide',
+    'trucks',
+    'uk_soccer',
+    'visual_acuity',
+    'von_bort',
+    'weldon_dice',
+    'wome
```
Comment (dustinvtran) [inline observations/r/__init__.py]: Remove this blankline

Comment (Arvinds-ds) [reply observations/r/__init__.py]: which blank line?

File: observations/r/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2308 @@
+"""[Observations.r](https://github.com/edwardlib/observations) provides
```
Comment (dustinvtran) [inline observations/r/__init__.py]: Actually, I just noticed this file is `r/__init__.py` and not `__init__.py`. You can remove this docstring. Alternatively, you can alter it to describe that this submodule consists of R data sets.

File: observations/r/__init__.py (hunk 2 above)
Comment (dustinvtran) [reply observations/r/__init__.py]: Oops misclick; the one on L2303.

File: observations/r/__init__.py (hunk 1 above)
Comment (Arvinds-ds) [reply observations/r/__init__.py]: fixed

File: observations/r/__init__.py (hunk 2 above)
Comment (Arvinds-ds) [reply observations/r/__init__.py]: fixed

File: observations/r/__init__.py (hunk 3 above)
Comment (Arvinds-ds) [reply observations/r/__init__.py]: fixed

**Comment by dustinvtran** [comment]
One more question: do you prefer to have imports via `from observations.r import abbey`?  Assuming the loading time is fine, what about changing `__init__.py` to support `from observations import abbey`? This keeps the module flat from the user's perspective but internally, it's organized hierarchically.

**Comment by Arvinds-ds** [comment]
I have no strong opinions but I prefer observations.r approach and keeping the main namespace clean from 1000 entries

**Comment by dustinvtran** [comment]
Got it. We'll keep it exposed hierarchically, at least for now. Will merge after travis passes.

**Comment by dustinvtran** [comment]
#19

**Pull request merged**